metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache-2.0'
 description       'Installs and configures haproxy'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '6.2.7'
+version           '6.2.8'
```

Comment:
Please revert this and instead add a changelog entry. We often roll up several changes into one release. 

Thanks!